src/app/components/LeftMenu/PartyCardList/Card/index.tsx

```diff
@@ -27,6 +27,7 @@ const Box = styled.div`
 `;
 
 const LeftWrapper = styled.div`
+    width:350px;
```

Comment:
it was worked fine before this commit...

this should be worked fine

I wonder reason